.github/workflows/main.yml

```diff
@@ -39,14 +39,14 @@ jobs:
         browser: [ "IE:11.0:Windows:7" , "Firefox:latest:OS X:Catalina" ]
     steps:
       - name: 'BrowserStack Env Setup'
-        uses: 'browserstack/github-actions/setup-env@master'
+        uses: 'browserstack/github-actions/setup-env@main'
```

Comment:
I thinj this needs to stay as `master` for now, because [the referenced repo ](https://github.com/browserstack/github-actions) does not have a `main` branche

Reply:
That's embarrassingly sloppy of me. Sorry! Will change these references back.

Reply:
haha, we've all done it!